### PR TITLE
Research source endpoint

### PR DESCRIPTION
**Pull request by simonarocco**
- added a research_fields boolean param to /records get and post endpoints
- in the get endpoint, this is a request param, in the post endpoint, this is part of the json body
- if set to True, all columns from research source table will be returned as well
- same filtering as before

### CODE REVIEW

File: app/utils/get_filtered_records.py [hunk 1]
```diff
@@ -23,10 +25,26 @@ def get_all_records():
...
                          'test_adj', 'specimen_type', 'test_type', 'population_group', 'url']
 
+        # Add columns from dashboard source to select statement
         fields_list = [DashboardSource.source_id]
         for field_string in field_strings:
             fields_list.append(getattr(DashboardSource, field_string))
 
+        # If research fields is True, add columns from research source to select statement
+        if research_fields:
+            research_source_cols = ['case_population', 'deaths_population', 'age_max', 'age_min', 'age_variation',
```
Comment (atatmaja) [inline app/utils/get_filtered_records.py:35]: do we want to put this in some config?

Comment (simonarocco) [reply app/utils/get_filtered_records.py:35]: i didnt put it in a config because the list of fields for dashboard source above was not in a config

**Review by atatmaja — APPROVED** [review]
Looks fine as long as it's been tested. One minor comment

File: app/utils/get_filtered_records.py (hunk 1 above)
Comment (atatmaja) [reply app/utils/get_filtered_records.py:35]: okay leaving it up to your judgement

**Pull request merged**